api/src/main/java/org/motechproject/nms/api/web/OpsController.java

```diff
@@ -175,5 +178,20 @@ public GetBookmarkResponse getBookmarkWithScore(@RequestParam(required = false)
         log("RESPONSE: /ops/getbookmark", String.format("bookmark=%s", ret.toString()));
         return ret;
     }
+
+    @RequestMapping(value = "/releaseNumber",
+            method = RequestMethod.POST,
+            headers = { "Content-type=application/json" })
+    @ResponseStatus(HttpStatus.OK)
+    @Transactional
+    public void releaseNumber(@RequestBody DeactivateSubscriptionContract deactivateSubscriptionContract) {
```

Comment:
We follow a request and reponse naming convention, rather than 'contract'. Please try to follow existing code naming conventions as much as possible